packages/datadog-instrumentations/src/jest.js

```diff
@@ -236,7 +236,6 @@ function getWrappedEnvironment (BaseEnvironment, jestVersion) {
             name: removeEfdStringFromTestName(testName),
             suite: this.testSuite,
             testSourceFile: this.testSourceFile,
-            runner: 'jest-circus',
```

Comment:
unrelated fix: this does not need to be reported from the instrumentation anymore, since the only runner is now `jest-circus`